driver/src/main.rs

```diff
@@ -185,6 +185,12 @@ struct Options {
 
     #[structopt(long, env = "ORDERBOOK_FILE", parse(from_os_str))]
     orderbook_file: Option<PathBuf>,
+
+    /// We calculate the maximum gas price cap based on the amount of earned fees from solution
+    /// submission. This factor is multiplied with the final result because we do not need to be
+    /// economically viable at the moment.
+    #[structopt(long, env = "GAS_PRICE_CAP_SUBSIDY_FACTOR", default_value = "2.0")]
```

Comment:
Still the question what the default value should be.

Reply:
I think we can leave it as is. We can always re-configure the deployment if needed.